tests/unit_tests/test_wave_06.py

```diff
@@ -228,16 +222,13 @@ def test_swap_best_by_category_no_match_is_false():
         their_priority="Clothing"
     )
 
-    raise Exception("Complete this test according to comments below.")
-    # *********************************************************************
-    # ****** Complete Assert Portion of this test **********
-    # *********************************************************************
-    # Assertions should check:
-    # - That result is falsy
-    # - That tai and jesse's inventories are the correct length
-    # - That all the correct items are in tai and jesse's inventories
+    assert result is False
+    assert len(tai.inventory) == 3
+    assert len(jesse.inventory) == 3 
+    assert tai.inventory == [item_a, item_b, item_c]
+    assert jesse.inventory == [item_d, item_e, item_f]
```

Comment:
Here, since the swaps fail, we'd expect items to remain in their same location. So a list comparison is more appropriate than the cases that did swap. Note, this means the tests assumem that internally, inventory is a list. While OK in this case, we still might prefer to write our tests not to assume this. Using `len` and `in` checks only requires that the inventory support asking its length and checking for membership, not forcing a plain python list.